PyInstaller/hooks/hook-_tkinter.py

```diff
@@ -219,6 +256,16 @@ def _collect_tcl_tk_files(hook_api):
 
     tcl_root, tk_root = _find_tcl_tk(hook_api)
 
+    # On macOS, we do not collect system libraries. Therefore, if system
+    # Tcl/Tk framework is used, it makes no sense to collect its data,
+    # either. In this case, _find_tcl_tk() will return None, None - either
+    # deliberately (we found the data paths, but ignore them) or not
+    # (starting with macOS 11, the data path cannot be found until shared
+    # library discovery is fixed).
+    if is_darwin and not tcl_root and not tk_root:
+        logger.info('Tcl/Tk not found on macOS system or system framework is used.')
+        return []
+
     # TODO Shouldn't these be fatal exceptions?
     if not tcl_root:
         logger.error('Tcl/Tk improperly installed on this system.')
```

Comment:
I suppose this is now a false positive on macOS11?

Reply:
It will be printed on macOS 11 with homebrew python, yes. But it will actually be correct on both accounts - the system framework is used, and therefore the shared libraries are currently not found.

It will also be printed on earlier macOS versions with older python.org builds (e.g., 32/64-bit 3.6.5) or homebrew python.